project.clj

```diff
@@ -1,14 +1,14 @@
-(defproject io.jesi/parent "4.6.0"
+(defproject io.jesi/parent "4.6.1"
   :description "Clojure and NPM parent package definitions"
   :url "https://github.com/jesims/parent#readme"
   :license {:name         "Eclipse Public License - v 1.0"
             :url          "http://www.eclipse.org/legal/epl-v10.html"
             :distribution :repo
             :comments     "same as Clojure"}
   :min-lein-version "2.9.1"
-  :managed-dependencies [[org.clojure/clojure "1.10.2"]
-                         [thheller/shadow-cljs "2.11.18"]
-                         [io.jesi/backpack "6.3.1"]
+  :managed-dependencies [[org.clojure/clojure "1.10.3"]
+                         [thheller/shadow-cljs "2.11.21"]
```

Comment:
Check `org.clojure/clojurescript` and other versions below.
Match versions used in shadow-cljs https://github.com/thheller/shadow-cljs/blob/master/project.clj

Reply:
They do.